### PR TITLE
Dynamic Bottom Sheet height

### PR DESCRIPTION
**Pull request by mfmurray**
## This is required to go into prod before

PR: https://gitlab.com/rownd/privacy-hub/-/merge_requests/225

### CODE REVIEW

**Comment by mfmurray** [comment]
Would we be interested in making the iOS Bottom Sheet backwards compatible? The bottom sheet would revert to the original slide up mode depending on the hub or an app-config param?

**Comment by mhamann** [comment]
Backward/forward compatibility would be ideal so the order of deployed components is flexible.

File: Sources/Rownd/Views/BottomSheetController.swift [hunk 1]
```diff
@@ -66,6 +66,25 @@ class BottomSheetController: UIViewController {
...
         self.controller = nil
     }
+    
+    func updateBottomSheetHeight(_ number: CGFloat) {
+        if let sheetController = sheetController {
+            guard let controller = self.controller else {
+                return
+            }
+            DispatchQueue.main.async {
+                let hubViewController = controller as! HubViewController
```
Comment (mhamann) [inline Sources/Rownd/Views/BottomSheetController.swift:76]: This should probably use conditional downcasting `as?` and check for `nil` just in case.

Comment (mfmurray) [reply Sources/Rownd/Views/BottomSheetController.swift]: Checks for nil have been added

**Pull request merged**